modules/explorer/info.go

```diff
@@ -38,6 +42,32 @@ func (e *Explorer) BlockFacts(height types.BlockHeight) (modules.BlockFacts, boo
 	return bf.BlockFacts, true
 }
 
+// LatestBlockFacts returns a set of statistics about the blockchain as they appeared
+// at the latest block height in the explorer's consensus set.
+func (e *Explorer) LatestBlockFacts() modules.BlockFacts {
+	var bf blockFacts
```

Comment:
If you want to mimic the existing paradigm, the closest approximation would be to create an unexported function `dbBlockFacts`:
```go
func (e *Explorer) dbGetBlockFacts(height types.BlockHeight) func(*bolt.Tx) error {
    // ...
}
```
Then `BlockFacts` and `LatestBlockFacts` would both call this function from within a bolt transaction, which serves the role of a mutex.